### DISCUSSION TITLE
Italian sentences that should be supported

### DISCUSSION THREAD

**Discussion by synesthesiam** [OHF-Voice/intents#370] [Supported Sentences]
@federicoodorizzi, @vilius365, @auanasgheps, @xraver - creating discussion for Italian translation here.

**Comment by ernst77** [comment]
I am not Italian sorry :) . IT !== LT

**Reply by synesthesiam** [reply]
Sorry, must have gotten you mixed up on my sheet!

**Comment by jjantoine** [comment]
Ciao amici italiani della community. Mi chiamo Antonio ed è un piacere collaborare con voi nello sviluppo del nostro amato Home Assistant.

Hi Italian community friends. I'm Antonio and I'm happy to contribute to the development of Home Assistant with you.

**Comment by auanasgheps** [comment]
Ciao, io e @xraver abbiamo creato la "base" delle frasi. 
Chiaramente è un lavoro limitato: parliamo di sole due persone, mentre sapete bene che la lingua italiana è molto vasta.

Se c'è bisogno di fare delle correzioni o volete proporre dei miglioramenti, parliamone qui. 
C'è bisogno di un minimo di dimestichezza con GitHub e con i file della repo. Se non volete addentrarvi, me ne occuperò io.

**Reply by iddiek** [reply]
A che punto siamo?

**Reply by auanasgheps** [reply]
Tu e altri avete fatto molte PR che hanno esteso il riconoscimento. Direi che abbiamo fatto un ottimo lavoro!

**Reply by xraver** [reply]
> A che punto siamo?

a me sinceramente sembra una delle lingue maggiormente supportata.
è anche vero che l'italiano rispetto a inglese è piu' complesso grammaticalmente ma secondo me abbiamo raggiunto un buon livello.
dovremmo testarlo non appena la funzione sarà presente in hass. e allora ci sara' spazio per ulteriori migliorie!!

I hope @balloob is happy about this :) It's a good way to support his great work and product.

**Comment by jokerigno** [comment]
ciao a tutti,

ieri è arrivato il mio Home Assistant Voice Preview Edition e ho provato a usarlo con pessimi risultati. Magari il problema è mio e della mia pessima pronuncia però mi sembrava giusto dare un feedback e mettermi a disposizione per quanto possa.

(ho provato i comandi sia via HAVPE che via iphone).

Ho anche provato a scrivere i messaggi e a farli sentire al voice preview ma il risultato non cambia

![image](https://github.com/user-attachments/assets/fd20fea0-7ba4-43f8-b0c6-768ad62fbcdd)

**Reply by auanasgheps** [reply]
per quello che posso dirti sembra essere un problema di trascrizione della voce.
Non è qualcosa che noi language leader possiamo fare, perché noi ci occupiamo di tradurre le frasi.

@synesthesiam this user is complaining that Italian Speech to Text recognition is poor - what should we check? By looking at the screenshot he's posted I can see many transcription errors. I don't think it's related to itents, but do you know who takes care of this?

**Reply by synesthesiam** [reply]
Can we find out which speech to text system they're using?

**Reply by auanasgheps** [reply]
> Can we find out which speech to text system they're using?

sorry for the dumb question, but is this system handled by the user or by HA Voice? I don't have such hardware and I use Assist only for written text, mainly to debug the intents.

**Reply by jokerigno** [reply]
> Can we find out which speech to text system they're using?

sure. I've made different tests. First one is running whisper and piper as addons in my VM. 

![image](https://github.com/user-attachments/assets/aa734ea9-69af-495a-bbe2-57bc54f4bf0b)

I also tried to use another whisper and piper running ad hoc in my unraid server (to bypass poor CPU of the host in which I run HA).

but nothing changed. Below config for the addons

![image](https://github.com/user-attachments/assets/2e901b5f-2d52-4e02-9470-d1fc287250ab)

![image](https://github.com/user-attachments/assets/d6f931d7-1ee7-4712-832c-f5be81967e29)
